src/test/java/net/sf/jabref/model/database/BibDatabaseTest.java

```diff
@@ -43,7 +48,8 @@ public void tearDown() {
      */
     @Test
     public void resolveStrings() throws IOException {
-        try (FileReader fr = new FileReader("src/test/resources/net/sf/jabref/util/twente.bib")) {
+        try (FileInputStream stream = new FileInputStream("src/test/resources/net/sf/jabref/util/twente.bib");
+                InputStreamReader fr = new InputStreamReader(stream, StandardCharsets.ISO_8859_1)) {
```

Comment:
I see now that the encoding is stated as UTF in the file. However since ISO is the first code page of UTF it is hard to actually determine the difference here... (And if I'm not wrong US_ASCII would also have worked for this file...)

I'll change it if there are other things to change as it is working and is sort of correct.